app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :users, class_name: 'User', foreign_key: 'users_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :posts, class_name: 'Post', foreign_key: 'posts_id'
 
```

Comment:
- [Optional] As suggested by the previous reviewer, in your models, kindly consider using the `after_save` method to update the counters in all your tables once a comment, like, or post is created. For instance, in the `Comment` model, you should add
```ruby
after_save :update_comment_counter
```
at the top of your model. This will update that post's `comment_counter` field so that the number of comments will be shown in real-time and doesn't have to be updated manually.

Reply:
update counter function calling done after_save event in Comment, Post and Like model.